src/new-components/storybook-helper.tsx

```diff
@@ -63,7 +63,7 @@ function times(count: number): number[] {
 
 function reusableBoxProps(): Partial<Record<keyof BoxProps, ReturnType<typeof selectWithNone>>> {
     return {
-        maxWidth: selectWithNone(['xsmall', 'small', 'medium', 'large', 'xlarge']),
+        maxWidth: selectWithNone(['xsmall', 'small', 'medium', 'large', 'xlarge', 'full']),
```

Comment:
What are your thoughts on adding the margins to this helper as well? I only ask because the `Box/Interactive Props Story` now includes padding values, but not margins which feels a little strange. Maybe it would make more sense to just add the margin value directly to the Interactive Props Story so it doesn't interfere with other ones in potentially unknown ways.

Reply:
Margin, unlike padding, is not added to what we call the "reusable box props". They are reusable in the sense that other higher-level layout components are allowed to have these props set on them directly as well. And it is not added on purpose: applying margin to Stacks, Columns, etc. may easily break them. TBH, even padding could break them, but not as easily. So, if at all, I may remove the ability to apply padding to anything other than a `Box`, instead of the other way around that you suggest.

Part of this is related to what I covered in [this video](https://www.dropbox.com/s/4cbsu5bum5ewgpl/2021-08%20design%20system%20and%20exceptionallySetClassName.mov?dl=0) that I shared a while back. Specifically the part where I talked towards the end about nesting layout components.